src/config/navigation/tabConfig.tsx

```diff
@@ -147,6 +148,7 @@ export const createDynamicTabConfig = (
         size={normalize(iconSize)}
         style={iconStyle}
       />
-    )
+    ),
+    tabBarLabelStyle
```

Comment:
please move it under `tabBarLabel` to have it close to the `tabBarLabel` logically